exporters/export_formatter/avro_export_formatter.py

```diff
@@ -0,0 +1,68 @@
+import json
+import six
+from fastavro.writer import Writer
```

Comment:
Using Avro is an optional feature, so this import shouldn't be done in the global namespace, it's best to do it inside of `_setup_writer`.

You should also add a section to [extras_require in setup.py](https://github.com/scrapinghub/exporters/blob/master/setup.py#L32-L48) like:

```
   'avro': ['fastavro'],
```

This way, when installing exporters you can install it with avro support doing:

    pip install exporters[avro]

Reply:
oh got it!  I love this approach.  Just remembered this is the approach I used while at SH. 

I will remove `fastavro` it from the requirements.txt . It makes it hard to develop exporters when you have to install all the dependencies :)